electronics/ohms_law.py

```diff
@@ -0,0 +1,46 @@
+def ohms_law(voltage: float, current: float, resistance: float) -> float:
```

Comment:
Need a URL to more info as discussed in CONTRIBUTING.md
https://en.wikipedia.org/wiki/Ohm%27s_law or other.